addOns/allinonenotes/src/main/java/org/zaproxy/zap/extension/allinonenotes/EventConsumerImpl.java

```diff
@@ -105,8 +106,7 @@ public void eventReceived(Event event) {
                 break;
             case HistoryReferenceEventPublisher.EVENT_REMOVED:
                 // This only gets fired if request is removed
-                LOGGER.debug("NOTE REMOVE EVENT recieved...");
-                LOGGER.debug(event.getParameters().toString());
+                LOGGER.debug("NOTE REMOVE EVENT recieved... {}", event.getParameters().toString());
```

Comment:
Could you also fix the typo in received? (In all debug messages.)

Reply:
Yup good catch